src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Data/DbDataMigrationService.cs

```diff
@@ -110,7 +110,7 @@ public async Task IngestAll(CancellationToken cancellationToken = default)
             await migrationService.LogMigration<Area>(dataKey, string.Empty, 5);
         }
 
-        if (migrationService.NeedMigration<Package>(dataKey, 5))
+        if (migrationService.NeedMigration<Package>(dataKey, 6))
         {
             await IngestPackage(options: options, cancellationToken: cancellationToken);
             await migrationService.LogMigration<Package>(dataKey, string.Empty, 5);
```

Comment:
Oppdater log migration versjon også

Reply:
Oppdaterte LogMigration for package. Jeg oppdaterte også IngestRole (og LogMigration). Burde jeg vente med dette siden Role api er i en annen branch?

Reply:
Nei, Stemmer jo at IngestRole også skal oppdateres her for at database endringene skal bli utført.

Men mulig denne endringen da gjør at du også trenger å oppdatere RoleMetadata API i etterkant (for å mappe ut IsAssignable feltet på API modellen?)